include/boost/uuid/name_generator.hpp

```diff
@@ -117,6 +116,38 @@ class name_generator {
     uuid namespace_uuid;
 };
 
-}} // namespace boost::uuids
+namespace ns {
+
+static BOOST_FORCEINLINE uuid dns() {
```

Comment:
Do not use `static` qualifiers in headers. This leads to ODR issues.

Reply:
I'll fix it with the next push - thanks!